webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/DataSourceController.scala

```diff
@@ -438,7 +438,7 @@ class DataSourceController @Inject()(
               name = datasetName,
               organization = organizationId,
               totalFileCount = 1,
-              filePaths = List.empty,
+              filePaths = Some(List.empty),
```

Comment:
maybe pass `None` now that this will default to an empty list anyway?

Reply:
sure sounds good :+1: